dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -359,19 +359,29 @@ def type_into_selector(input_text, selector)
   @browser.execute_script 'dashboard.clientState.trackProgress(true, 1, 100, "hourofcode", appOptions.serverLevelId)'
 end
 
-Then /^I verify progress in the header of the current page is "([^"]*)" for level (\d+)/ do |test_result, level|
+Then /^I wait to see the progress header$/ do
   steps %{
     And I wait to see ".header_level_container"
-    And I wait for 10 seconds
+    And I wait for 5 seconds
+  }
+end
+
+Then /^I verify progress in the header of the current page is "([^"]*)" for level (\d+)/ do |test_result, level|
+  steps %{
     And element ".header_level_container .react_stage a:nth(#{level.to_i - 1}) :first-child" has css property "background-color" equal to "#{color_for_status(test_result)}"
   }
 end
 
-Then /^I verify progress in the drop down of the current page is "([^"]*)" for stage (\d+) level (\d+)/ do |test_result, stage, level|
+Then /^I open the progress drop down of the current page$/ do
   steps %{
     Then I click selector ".header_popup_link"
     And I wait to see ".user-stats-block"
     And I wait for 10 seconds
+  }
```

Comment:
Both of these are coupled with a 'wait for element' step that is presumably the thing we're waiting to see anyway.  I'm not 100% sure why the extra wait exists in the first place, but Brendan isn't around to clarify.  I can research further on this though.